docker-compose.yml

```diff
@@ -18,17 +18,16 @@
 # You can run tests like this:
 #
 # ```
-# docker-compose run --rm test bundle exec cucumber features/other_a/autocomplete.feature
+# docker compose run --rm test bundle exec cucumber features/other_a/autocomplete.feature
 # ```
 
-version: "3"
```

Comment:
Triggers a deprecation warning now, and a quick Google search suggests it's fine to just remove it.